pkg/backplane/status/status.go

```diff
@@ -0,0 +1,45 @@
+package status
+
+import "fmt"
+
+const (
+	// Running means the sonobuoy run is still in progress
+	Running string = "running"
+	// Complete means the sonobuoy run is complete
+	Complete string = "complete"
+	// Failed means one or more plugins has failed and the run will not complete successfully
+	Failed string = "failed"
+)
+
+// Plugin represents the current status of an individual plugin
+type Plugin struct {
+	Plugin string `json:"plugin"`
+	Node   string `json:"node"`
+	Status string `json:"status"`
+}
+
+// Status represents the current status of a Sonobuoy run
+type Status struct {
```

Comment:
`status.Status.Status` isn't... great. Any ideas for breaking this up?

Reply:
update.Run.Status ?